chrome/content/zotero/bibliography.js

```diff
@@ -195,10 +212,11 @@ var Zotero_File_Interface_Bibliography = new function() {
 		
 		// Change label to "Citation" or "Note" depending on style class
 		if(document.getElementById("citations")) {
+			var label = "";
```

Comment:
Doesn't matter too much, but better to use ``let`` instead of ``var`` when a variable is only needed within a given block.